### PR TITLE
Issue #4581 - remove unnecessary javadoc on Override methods

### PR DESCRIPTION
**Pull request by lachlan-roberts**
**Issue #4581**

Remove unnecessary javadoc on `@Override` methods.

### CODE REVIEW

File: jetty-infinispan/infinispan-common/src/main/java/org/eclipse/jetty/session/infinispan/InfinispanSessionDataStore.java [hunk 1]
```diff
@@ -82,9 +82,6 @@ public void setQueryManager(QueryManager queryManager)
         _queryManager = queryManager;
     }
 
-    /**
-     * @see org.eclipse.jetty.server.session.SessionDataStore#load(String)
```
Comment (lachlan-roberts) [inline jetty-infinispan/infinispan-common/src/main/java/org/eclipse/jetty/session/infinispan/InfinispanSessionDataStore.java:86]: see `SessionDataStore.load` in javadoc for `doStart`

File: jetty-io/src/main/java/org/eclipse/jetty/io/ByteArrayEndPoint.java [hunk 2]
```diff
@@ -465,9 +456,6 @@ public void reset()
         super.reset();
     }
 
-    /*
-     * @see org.eclipse.io.EndPoint#getConnection()
```
Comment (lachlan-roberts) [inline jetty-io/src/main/java/org/eclipse/jetty/io/ByteArrayEndPoint.java:469]: see `EndPoint.getConnection()` in javadoc for `getTransport()`

File: tests/test-distribution/src/main/java/org/eclipse/jetty/tests/distribution/DistributionTester.java [hunk 3]
```diff
@@ -479,9 +479,6 @@ private void stopConsoleStreamers()
             consoleStreamers.forEach(ConsoleStreamer::stop);
         }
 
-        /**
-         * @see #destroy()
```
Comment (lachlan-roberts) [inline tests/test-distribution/src/main/java/org/eclipse/jetty/tests/distribution/DistributionTester.java:483]: see `destroy()` in javadoc for `close()`

**Review by lachlan-roberts — COMMENTED** [review]
Scrolled through the changes and found a few weird things.

**Review by gregw — APPROVED** [review]
Good work!

I'm wondering if we should do the same in 9?

**Review by olamy — APPROVED** [review]

**Pull request merged**